library/Claim.class.php

```diff
@@ -265,6 +266,17 @@ function Claim($pid, $encounter_id) {
       "ORDER BY forms.date";
     $this->billing_options = sqlQuery($sql);
 
+    if ($GLOBALS['claim_type'] =='1' || $GLOBALS['claim_type'] =='2') {
```

Comment:
Is there a global that toggles between either professional or institutional billing and prevents you from billing both types?